agent/pool/conn.go

```diff
@@ -28,19 +30,19 @@ func (t RPCType) ALPNString() string {
 const (
 	// keep numbers unique.
 	RPCConsul      RPCType = 0
-	RPCRaft                = 1
-	RPCMultiplex           = 2 // Old Muxado byte, no longer supported.
-	RPCTLS                 = 3
-	RPCMultiplexV2         = 4
-	RPCSnapshot            = 5
-	RPCGossip              = 6
+	RPCRaft        RPCType = 1
```

Comment:
Since an `iota` isn't used here, only `RPCConsul` had the type `RPCType`. Something I did in this PR surfaced this issue so I just fixed it in place.